src/core/xds/grpc/xds_client_grpc.cc

```diff
@@ -233,6 +250,11 @@ absl::StatusOr<RefCountedPtr<GrpcXdsClient>> GrpcXdsClient::GetOrCreate(
   // instance for the channel or server instead of using the global instance.
   absl::optional<absl::string_view> bootstrap_config = args.GetString(
       GRPC_ARG_TEST_ONLY_DO_NOT_USE_IN_PROD_XDS_BOOTSTRAP_CONFIG);
+  std::string channel_default_authority =
```

Comment:
Doesn't look like this is used anywhere, so I don't think it's needed.

Reply:
Done